### ISSUE TITLE
Parsing error with Invalid character 0xd

### ISSUE THREAD

**Issue by vinsha2k** [CESNET/libyang#2320]
Hi, 
libyang-2.1.55 - I am facing issue where client is not able to parse get-schema reply from server, getting error as "ly ERROR: Invalid character 0xd. (Line number 76.)". below is the snippet of schema where parse error occurred.
```
      leaf object-unit-id {
        type uint8;
        description
          "Antenna path id (counting from 0) is used for the object-unit-id for the 
           vendor-specific-measurement-result, for which object-unit is VSWR, TX_OUTPOWER or TX_INPOWER 
         
        For the vendor-specific-measurement-result where object-unit is RSSI, object-unit-id is encode the followin way: 
        bit 7..4:\tAntenna path (counting from 0) 
        bit 3..0:    Carrier Id per path (counting from 0)";
      }
```
```
      leaf object-unit-id {
        type uint8;
        description
          "Antenna path id (counting from 0) is used for the object-unit-id for the
           vendor-specific-measurement-result, for which object-unit is VSWR, TX_OUTPOWER or TX_INPOWER   <==== Parse error occurs here

                   For the vendor-specific-measurement-result where object-unit is RSSI, object-unit-id is encode the followin way:
                   bit 7..4:\tAntenna path (counting from 0)
                   bit 3..0:    Carrier Id per path (counting from 0)";
      }
```
Please help what could cause this.

**Comment by michalvasko** [comment]
I am pretty sure that if you update *libyang*, it will work. If not, attach the problematic YANG module so that I can reproduce the issue.

**Comment by vinsha2k** [comment]
Thanks Michal, please find the yang attached.
[vendor-specific-performance-management@2021-08-03.zip](https://github.com/user-attachments/files/17699205/vendor-specific-performance-management%402021-08-03.zip)

Also if you could suggest fix specific for this issue on libyang-2.1.55 version, that would help.

**Comment by michalvasko** [comment]
I could not reproduce the issue with the latest versions of the projects. That is the solution we provide, you can update your versions and it will work. If you only want a patch, you have to find the relevant commit(s) yourself. It should be changes in *libyang* connected to newline handling.
